src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -195,9 +200,9 @@ public void process(RestRequest request, RestChannel channel, RestFilterChain ch
 					logger.debug("Handling Request...");
 					logger.debug("Evaluating request for user '{}' with a {} token", user,
 							(StringUtils.isNotEmpty(token) ? "non-empty" : "empty"));
-					logger.debug("Cache has user: {}", cache.hasUser(user));
+					logger.debug("Cache has user: {}", cache.hasUser(user, token));
```

Comment:
Not sure what you mean.  The one bracket pair `{}` is for the boolean return value of `cache.hasUser`